ganga/GangaDirac/__init__.py

```diff
@@ -115,12 +115,34 @@ def loadPlugins(config=None):
     from .Lib import Files
 
 def postBootstrapHook():
+    from GangaCore.GPIDev.Lib.Config.Config import getConfig#, config_file_as_text
+    from GangaCore.Runtime.bootstrap import GangaProgram
+    regenerate = False
+
     dirac_conf = getConfig('DIRAC')
     if not dirac_conf['DiracEnvSource'] and not dirac_conf['DiracEnvJSON']:
-        return
+        source = input("Please enter the location of your Dirac UI bashrc file.")
+        if not source:
+            logger.warning("No location specified, Dirac plugin will likely not work!")
+            return
+        dirac_conf.setGangarcValue("DiracEnvSource", source)
+        regenerate = True
+#            config_file_as_text(False)
+
+    dirac_proxy = getConfig("defaults_DiracProxy")["group"]
+    if not dirac_proxy["group"] or dirac_proxy["group"] == "None":
+        group = input("Which group would you like to use for you proxy (e.g. <VO>_user)")
```

Comment:
Again, is it not better to get the user to write a correct `.gangarc` file and then abort. Maybe we can collate everything that is wrong and then write a single message (to avoid that user will have to start Ganga 4 times and each time be told that something new is missing.

Reply:
The idea here is that it prompts the user and then writes the gangarc options for them. This code shouldn't need a ganga restart.

Reply:
I see. That is fine as well.